Make config fails on clean build envivronment

<!-- Please describe your issue in English. -->

#### Can be reproduced ?
yes

#### Steps:
Run make config on a clean environment
$ make config
go build -o gen_config cmd/generate-config/main.go cmd/generate-config/config_template.go
go: github.com/BurntSushi/toml@v0.3.1: missing go.sum entry; to add it:
	go mod download github.com/BurntSushi/toml
make: *** [Makefile:2: config] Error 1


#### Expected behavior:
Successfully compelte config

#### Actual behavior:
n/a

#### Environment:
- Version or commit-id (e.g. v0.1.0 or 8b23a93): v0.1.0 df9b134970edc6e4abcfed5281b8a1b8500a879b
- Hardware parameters: n/a
- OS type: CentOS 8
- Others:

#### Configuration file:
n/a

#### Additional context:
- Error message from client:
- Server log:
- Other information:

--- a/Makefile
+++ b/Makefile
@@ -1,4 +1,5 @@
 config: cmd/generate-config/main.go cmd/generate-config/config_template.go cmd/generate-config/system_vars_def.toml
+	go mod tidy
 	go build -o gen_config cmd/generate-config/main.go cmd/generate-config/config_template.go
 	./gen_config cmd/generate-config/system_vars_def.toml
 	mv -f cmd/generate-config/system_vars_config.toml .

